### PR TITLE
Create adviser configmap with all the configuration options

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/thoth-application/issues/1086

### CODE REVIEW

File: core/base/configmaps.yaml [hunk 1]
```diff
@@ -13,6 +13,15 @@ metadata:
 ---
 kind: ConfigMap
 apiVersion: v1
+metadata:
```
[the remark this answers is not in the document]
Comment (fridex) [reply core/base/configmaps.yaml]: Adjusted in e5a5d46176b9600562012571a40081641ec0c9de. PTAL 👍🏻

File: adviser/overlays/aws-prod/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (harshad16) [inline adviser/overlays/aws-prod/configmap.yaml:1]: we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/aws-prod/kustomization.yaml#L10

File: adviser/overlays/ocp4-stage/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+kind: ConfigMap
```
Comment (harshad16) [inline adviser/overlays/ocp4-stage/configmap.yaml:2]: we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/ocp4-stage/kustomization.yaml#L10

File: adviser/overlays/moc-prod/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+kind: ConfigMap
```
Comment (harshad16) [inline adviser/overlays/moc-prod/configmap.yaml:2]: we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/moc-prod/kustomization.yaml#L10

File: adviser/overlays/test/configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+kind: ConfigMap
```
Comment (harshad16) [inline adviser/overlays/test/configmap.yaml:2]: we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/test/kustomization.yaml#L8

**Review by harshad16 — CHANGES_REQUESTED** [review]
thanks for the previous changes.
Need a little more adjustment.
can you add the `configmap` in the overalys to the kustomization.yaml

File: adviser/overlays/ocp4-stage/configmap.yaml (hunk 3 above)
Comment (fridex) [reply adviser/overlays/ocp4-stage/configmap.yaml:2]: Done in 1d1b34d0, thanks for spotting this 👍🏻

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**